src/Compilers/CSharp/Portable/Emitter/Model/ParameterSymbolAdapter.cs

```diff
@@ -29,11 +29,11 @@ bool Cci.IParameterTypeInformation.IsByReference
             }
         }
 
-        bool Cci.IParameterTypeInformation.HasByRefBeforeCustomModifiers
+        ushort Cci.IParameterTypeInformation.CountOfCustomModifiersPrecedingByRef
```

Comment:
I do not remember seen any specified limit, but I believe it is reasonable for us to not support signatures with amount of custom modifiers that doesn't fit into ushort. We can even go to byte, but one might argue that having more than 255 modifiers is something that could happen in the wild.